TESTS/usb_device/msd/main.cpp

```diff
@@ -0,0 +1,533 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdio.h>
+#include <string.h>
+#include "greentea-client/test_env.h"
+#include "unity/unity.h"
+#include "utest/utest.h"
+
+#include "mbed.h"
+#include "USBMSD.h"
+#include "TestUSBMSD.h"
+
+#include "HeapBlockDevice.h"
+#include "MBRBlockDevice.h"
+#include "BlockDevice.h"
+#include "FATFileSystem.h"
+
+
+#if !defined(DEVICE_USBDEVICE) || !DEVICE_USBDEVICE
+#error [NOT_SUPPORTED] USB Device not supported for this target
+#endif
+
+#ifdef MIN
+#undef MIN
+#endif
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+
+#define DEFAULT_BLOCK_SIZE  512
+#define HEAP_BLOCK_DEVICE_SIZE (128 * DEFAULT_BLOCK_SIZE)
+#define MIN_HEAP_SIZE (HEAP_BLOCK_DEVICE_SIZE + 6144)
+
+
+/* TODO:
```

Comment:
Did you want to take action on this TODO?

Reply:
Sure, as soon as it will be supported in USBMSD class